docs/libraries/eigensolvers/dlaf.md

```diff
@@ -0,0 +1,57 @@
+# DLA-Future
+
+[DLA-Future] is a distributed linear algebra library implemented using the C++ `std::execution` proposal [P2300] as provided by [pika].
```

Comment:
This could perhaps also simply refer to "implemented using the C++26 `std::execution` library (link to https://eel.is/c++draft/#exec) as provided by [pika]"? P2300 is, however, good as a reference too, and I'm aware this is still what we refer to in first paragraph on pikacpp.org.

Reply:
P2300 is also referred in the `pika`-specific paragraph. I apply your suggestion at the top level and re-phrase the `pika`-specific paragraph.